Dockerfile

```diff
@@ -22,5 +22,15 @@ RUN mkdir /leonardo
 COPY ./leonardo*.jar /leonardo
 COPY --from=helm-go-lib-builder /helm-go-lib-build/helm-scala-sdk/helm-go-lib /leonardo/helm-go-lib
 
+# Install the Helm3 CLI client from the source because installing it via the RHEL package managing didn't work
```

Comment:
I think installing from script is a bit different from installing from source :)

Reply:
picky picky 😛 
j/k you're right - corrected